### PR TITLE
Housenumber check for LU and CH

### PR DESCRIPTION
**Pull request by grischard**
Hello,

This adds regex checks for house numbers in two countries, and refactors the code to make the addition of new countries easier.

Additionally, this cleans up the plugin's code according to PEP8.

@ltog helped out with the original idea, the implementation, and the Swiss regex.

This is my first real PR to Osmose, please be critical and generous in your feedback :)

### CODE REVIEW

File: plugins/TagFix_Housenumber.py [hunk 1]
```diff
@@ -26,20 +26,32 @@ class TagFix_Housenumber(Plugin):
...
     def node(self, data, tags):
         err = []
-        if self.CountryHousenumberWithoutNumber:
-          return []
-        if "addr:housenumber" in tags and (len(tags["addr:housenumber"]) == 0 or not (
-            tags["addr:housenumber"][0].isdigit() or
-            (self.CountryCZ and tags["addr:housenumber"].startswith('ev.') and tags["addr:housenumber"][3].isdigit())
-            )):
+        if self.Country in ('RU', 'BG'):  # Countries with no house numbers
+            return []
```
Comment (frodrigo) [inline plugins/TagFix_Housenumber.py]: For 'RU' and 'BG' we have now a now a `not_for`
https://github.com/osm-fr/osmose-backend/blob/master/plugins/Source.py#L28

File: plugins/TagFix_Housenumber.py [hunk 2]
```diff
@@ -26,20 +26,32 @@ class TagFix_Housenumber(Plugin):
...
+        if "addr:housenumber" in tags and (len(tags["addr:housenumber"]) == 0 or not (housenumberRegexByCountry[self.Country].match(tags["addr:housenumber"]))):
```
Comment (frodrigo) [inline plugins/TagFix_Housenumber.py]: Make sure `housenumberRegexByCountry[self.Country]` exists

Comment (grischard) [inline plugins/TagFix_Housenumber.py]: I'm using a defaultdict to always have a value:
```
housenumberRegexByCountry = defaultdict(re.compile("^[1-9]"))
```

File: plugins/TagFix_Housenumber.py (hunk 1 above)
Comment (grischard) [inline plugins/TagFix_Housenumber.py]: I've used `not_for` to replace this check.

File: plugins/TagFix_Housenumber.py (hunk 2 above)
Comment (frodrigo) [inline plugins/TagFix_Housenumber.py]: ok

File: plugins/TagFix_Housenumber.py [hunk 3]
```diff
@@ -123,3 +123,49 @@ class father:
...
+
+        assert not a.way(None, {"addr:housenumber": "ev.387"}, None)  # In CZ
+
+
+    def test_LU(self)
+        a = TagFix_Housenumber(None)
+
+        class _config:
+            options = {"country": "CH"}
+
```
Comment (frodrigo) [inline plugins/TagFix_Housenumber.py]: Not "LU" here ?

Comment (grischard) [inline plugins/TagFix_Housenumber.py]: Fixed!

**Comment by frodrigo** [comment]
I have compiled all commits in one and fixed it:
https://github.com/frodrigo/osmose-backend/commit/6e2bb4c208f1108154bb8d94aedb27e84aa33ac3

Note: plugins, unlike osmosis analysers, are easy to run and test, use `nosetests TagFix_Housenumber.py`

**Pull request closed without merging**